augurpy/estimators.py

```diff
@@ -21,13 +21,16 @@ def _raise_exception(exception_message: str):
     raise Exception(exception_message)
 
 
+params = Params()
```

Comment:
Not very pretty, only because of: B008 Do not perform function calls in argument defaults.  The call is performed only once at function definition time. All calls to your function will reuse the result of that definition-time function call.  If this is intended, assign the function call to a module-level variable and use that variable as a default value.

Reply:
We don't have extremely strict code quality requirements here. Some super annoying flake8 errors can also be added to the .flake8 file or inline in the code with `#noqa: XXX`.

Just saying if you think that you'd rather switch this warning off. This particular one however can lead to serious issues (just unlikely in augurpy because augurpy is not complex software).

Reply:
Then we keep it as it is now! Also had not seen a lot of module-level variables and was not sure if this was the way to do it:)